index.html

```diff
@@ -63,6 +65,55 @@ <h1>
 
     <div class="container">
 
+        <section class="main">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-3 main_one">
+                        <div class="main_categories">
+                            <div class="main_categories_all">
+                                <i class ="fa fa-bars"></i>
+                                <span> All categories</span>
+                            </div>
+                            <ul>
+                                <li><a href=#></a></li>
```

Comment:
hm...what this big list of empty links is for?

Reply:
I think you can get rid of these for now and of will be needed you can always add those.